roles/ceph-common/tasks/os_tuning.yml

```diff
@@ -22,4 +22,5 @@
     value={{ item.value }}
     state=present
     sysctl_file=/etc/sysctl.conf
+    ignoreerrors=yes
```

Comment:
I guess you meant: ignore_errors: True
This should appear below the with_items statement.